conduit/pipeline/pipeline.go

```diff
@@ -306,11 +306,11 @@ func (p *pipelineImpl) Init() error {
 		if err != nil {
 			return fmt.Errorf("Pipeline.Init(): could not make %s config: %w", p.cfg.Importer.Name, err)
 		}
-		err = (*p.importer).Init(p.ctx, *p.initProvider, pluginConfig, importerLogger)
+		err = (p.importer).Init(p.ctx, *p.initProvider, pluginConfig, importerLogger)
```

Comment:
Are the parens still needed?
```suggestion
		err = p.importer.Init(p.ctx, *p.initProvider, pluginConfig, importerLogger)
```

Reply:
f507ba58379a85661a19bdc51d6a2eb57ae40a78